source/plugin/wgen/modifiers.rst

```diff
@@ -63,15 +64,14 @@ apply your generator changes to nether worlds.
 Registering a WorldGeneratorModifier
 ====================================
 
-Now that you have created our modifier, you need to register it. A good time to do this is during the ``INITIALIZATION``
-State. To register it, simply call :javadoc:`GameRegistry#register(Class, T)` with ``WorldGeneratorModifier.class`` as
-the first argument and your modifier as the second.
+Now that you have created our modifier, you need to register it. A good time to do this is during the ``GAMEREGISTRY``
+Event. To register it, simply listen to :javadoc:`GameRegistryEvent` then register from there.
```

Comment:
I would rephrase it like this: (including some additional cross references to other pages)


````text
Now that you have created the modifier, all you have to do is register it in the :javadoc:`GameRegistry`. To do this you need to create a :doc:`listener </plugin/event/listeners>` for the :javadoc:`GameRegistryEvent.Register<WorldGeneratorModifier>` event and register your instance there, as shown in the following example:
````


With that changes you only have to change the javadoc import at the top from `GameRegistryEvent` to `GameRegistryEvent.Register`.